Documentation/update-modules-v3-file-api.md

```diff
@@ -0,0 +1,227 @@
+Update modules v3 protocol
+==========================
+
+Update modules are executables that are placed in `/usr/lib/mender/modules/v3`
+directory, where `v3` is a reference to the version of the protocol. Mender will
+look in the directory with the same version as the version of the artifact being
+processed.
+
+
+States and execution flow
+-------------------------
+
+Update modules are modelled after the same execution flow as state scripts, and
```

Comment:
Link state scripts to https://docs.mender.io/artifacts/state-scripts ?

Reply:
Yep!